apollo-router/src/plugins/telemetry/metrics/apollo.rs

```diff
@@ -102,13 +104,31 @@ impl Config {
         tracing::debug!(endpoint = %endpoint, "creating Apollo OTLP metrics exporter");
         let mut metadata = MetadataMap::new();
         metadata.insert("apollo.api.key", key.parse()?);
+        let exporter = MetricsExporterBuilder::Tonic(
+            opentelemetry_otlp::new_exporter()
+                .tonic()
+                .with_endpoint(endpoint.as_str())
+                .with_timeout(batch_processor.max_export_timeout)
+                .with_metadata(metadata)
+                .with_compression(opentelemetry_otlp::Compression::Gzip),
```

Comment:
is this something we want to make configurable?

Reply:
I don't think so for apollo metrics, but could be persuaded if we think there is a good reason to.

For user otlp metrics we should absolutely add a configuration option. Happy to open a new ticket for that.

Reply:
https://github.com/apollographql/router/issues/3916